src/myrtlespeech/model/deep_speech_1.py

```diff
@@ -76,7 +76,7 @@ def __init__(
 
         self.use_cuda = torch.cuda.is_available()
 
-        self._relu_clip = relu_clip
+        self._relu_clip = float(relu_clip)
```

Comment:
If the user passes an int instead of a float (as may occur if initialising the `DeepSpeech1` class directly (rather than with the protobufs), the model won't export to onnx as the `Hardtanh` max and min values must be floats.